packages/pangea-sdk/pangea/services/vault/models/common.py

```diff
@@ -259,3 +262,60 @@ class UpdateRequest(APIRequestModel):
 
 class UpdateResult(APIRequestModel):
     id: str
+
+
+class JWTGetRequest(APIRequestModel):
+    id: str
+    version: Optional[str] = None
+
+
+class JWKHeader(PangeaResponseResult):
```

Comment:
I'm not sure if we need to interpret this, or just leave it as a dict... the user probably has a library to manipulate them